src/components/Dialog/BaseDialog/BaseDialog.tsx

```diff
@@ -18,26 +18,31 @@ export const BaseDialog: FC<BaseDialogProps> = React.forwardRef(
             actionButtonOneProps,
             actionButtonTwoProps,
             actionButtonThreeProps,
+            actions,
+            actionsClassNames,
+            body,
+            bodyClassNames,
+            bodyPadding = true,
+            closable = true,
             closeButtonProps,
             closeIcon = IconName.mdiClose,
-            parent = document.body,
-            visible,
-            onClose,
-            maskClosable = true,
-            onVisibleChange,
-            height,
-            width,
-            zIndex,
+            dialogClassNames,
+            dialogWrapperClassNames,
             header,
+            headerButtonProps,
             headerClassNames,
-            body,
-            bodyClassNames,
-            actions,
-            actionsClassNames,
-            dialogWrapperClassNames,
-            dialogClassNames,
+            headerIcon = IconName.mdiArrowLeftThick,
```

Comment:
Can we have no headerIcon by default as this will break existing users functionalty

Reply:
Hold-on - this is a new prop - how will it break existing apps?

Reply:
Cause if we show it by default, it will start showing up in places where the developer had not planned for

Reply:
This con doesn't show unless the headerIconProps are defined.